messages.pot

```diff
@@ -17,38 +17,19 @@ msgstr ""
 "Content-Transfer-Encoding: 8bit\n"
 "Generated-By: Babel 2.10.3\n"
 
-#: app.py:122
+#: app.py:117
 msgid "Apply for funding to save an asset in your community"
 msgstr ""
 
-#: frontend/templates/footer.html:9
-msgid "Privacy"
+#: frontend/magic_links/templates/email.html:19
+msgid "Email address "
 msgstr ""
 
-#: frontend/templates/footer.html:14
-msgid "Cookies"
-msgstr ""
-
-#: frontend/templates/footer.html:18
-msgid "Accessibility Statement"
-msgstr ""
-
-#: frontend/templates/footer.html:22
-msgid "Contact us"
-msgstr ""
-
-#: frontend/templates/header.html:34
-msgid "beta"
-msgstr ""
-
-#: frontend/templates/header.html:36
-msgid "This is a new service - your "
-msgstr ""
-
-#: frontend/templates/header.html:36
-msgid "feedback"
```

Comment:
These strings shouldn't be deleted. We need to make sure the .pot file includes these strings.

Reply:
The babel config path to .html files was incorrect so I had to update the path, probably at the time it deleted all old strings. I will add these back on

Reply:
This is .pot file. Babel config had incorrect file path to read .html files so had to update the path, probably at that time old strings got deleted from .pot file. However the translations exist for these texts in the .po file so nothing is actually missing.
 
<img width="1792" alt="Screenshot 2022-11-15 at 16 15 17" src="https://user-images.githubusercontent.com/80714392/201970574-60cb17d5-37eb-4d7d-ab23-8fa3df5625bd.png">